### DISCUSSION TITLE
Add a working branch with latest improvements

### DISCUSSION THREAD

**Discussion by sc0w** [JamesIves/github-pages-deploy-action#1720] [Ideas]
If I use the default branch `dev` it fails:

```
Run JamesIves/github-pages-deploy-action@dev
Error: File not found: '/home/runner/work/_actions/JamesIves/github-pages-deploy-action/dev/lib/main.js'
```

I think maybe this is the expected behavior, but I would like to use always the same branch with latest improvements, maybe is possible to add new branch which always works? maybe with the name `latest` ?

**Comment by JamesIves** [comment]
The release cycle for this project is super tight, often times when a change is made to `dev` it very shortly after that gets moved to the next release (with the exception of some Dependabot updates). Pointing to a dev could branch have some negative side effects for you as if a bug is merged it means your workflows will automatically inherit it without any change from you. 

In my opinion, it's best to point to a specific tagged version, but if you _really_ want cutting edge releases, you can point to the `@releases/v4` branch.

**Comment by sc0w** [comment]
the question here is to avoid to edit again and again when new release is done, maybe latest release can have the same tag? `latest` for example?

**Reply by JamesIves** [reply]
You can use either `JamesIves/github-pages-deploy-action@v4` for that (the major version tag is always updated with the latest change post-release), or leverage Dependabot to keep things updated.

**Reply by sc0w** [reply]
is not possible to add `lib` folder to `dev` branch?

**Reply by JamesIves** [reply]
It's possible, but not something I'll be investing time in right now.
